frontend/src/metabase/public/lib/code.ts

```diff
@@ -16,21 +16,25 @@ import {
 export const getEmbedClientCodeExampleOptions =
   (): ClientCodeSampleConfig[] => [
     {
+      id: "pug",
```

Comment:
this is the new value that'll be used in Select and also sent to analytics instead of using `name`